pkg/providers/amifamily/bootstrap/eksbootstrap.go

```diff
@@ -119,6 +119,14 @@ func (e EKS) isIPv6() bool {
 	if e.KubeletConfig == nil || len(e.KubeletConfig.ClusterDNS) == 0 {
```

Comment:
Bottlerocket and AL2023 also have functionality for this. Can we also add support for this to those AMIFamily as well?